src/main/java/org/eclipse/yasson/YassonConfig.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2019 IBM and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+package org.eclipse.yasson;
+
+import java.util.Map;
+
+import javax.json.bind.JsonbConfig;
+import javax.json.bind.serializer.JsonbSerializer;
+
+/**
+ * Custom properties for configuring Yasson outside of the specification {@link javax.json.bind.JsonbConfig} scope.
+ */
+public class YassonConfig extends JsonbConfig {
+    
+    /**
+     * @see #withFailOnUnknownProperties(boolean)
+     */
+    public static final String FAIL_ON_UNKNOWN_PROPERTIES = "jsonb.fail-on-unknown-properties";
+
+    /**
+     * @see #withUserTypeMapping(java.util.Map)
+     */
+    public static final String USER_TYPE_MAPPING = "jsonb.user-type-mapping";
+    
+    /**
+     * @see #withZeroTimeParseDefaulting(boolean)
+     */
+    public static final String ZERO_TIME_PARSE_DEFAULTING = "jsonb.zero-time-defaulting";
+
+    /**
+     * @see #withNullRootSerializer(javax.json.bind.serializer.JsonbSerializer)
+     */
+    public static final String NULL_ROOT_SERIALIZER = "yasson.null-root-serializer";
+
+    /**
+     * @see #withEagerParsing(Class...)
+     */
+    public static final String EAGER_PARSE_CLASSES = "yasson.eager-parse-classes";
+    
+    /**
+     * Property used to specify behaviour on deserialization when JSON document contains properties
+     * which doesn't exist in the target class. Default value is 'false'.
```

Comment:
Before we had the default value incorrectly documented as `true`, changed it to `false` here